tests/test_client.py

```diff
@@ -58,6 +66,50 @@ def test_correct_new_auth0(self):
             auth0=new_auth0,
         )
 
+    @pytest.mark.parametrize(
+        "env, core_endpoint_expected",
+        [
+            ('prod', 'https://apis.pasqal.cloud/core-fast'),
+            ('preprod', 'https://apis.preprod.pasqal.cloud/core-fast'),
+            ('dev', 'https://apis.dev.pasqal.cloud/core-fast')
+        ]
+    )
+    def test_good_env(self, env, core_endpoint_expected):
+        sdk = SDK(
+            project_id=self.project_id,
+            username=self.username,
+            password=self.password,
+            env=env
+        )
+        assert sdk._client.endpoints.core == core_endpoint_expected
+
+    def test_bad_env(self):
```

Comment:
`test_init_sdk_with_unknown_env`

On a side note this seems to be in the `TestAuthSuccess` class which doesn't seem appropriate. We may want to either rename the test class or move this out of it